src/query/api/v1/handler/prometheus/common.go

```diff
@@ -347,6 +375,12 @@ func renderDefaultTagCompletionResultsJSON(
 		jw.BeginObjectField("values")
 		jw.BeginArray()
 		for _, value := range tag.Values {
```

Comment:
So I'm assuming it is intentional here that we still write the key/value field/array even once over limit, and that the arrays are just empty?